app/code/community/Bolt/Boltpay/OrderCreationException.php

```diff
@@ -153,7 +158,7 @@ public function __construct(
             $dataValues = array($message."\nSupplied error:\n$originalJson");
         }
 
-        foreach( $dataValues as $index => $value ) {
+        foreach($dataValues as $index => $value ) {
```

Comment:
This really bothers me... Either there's a space or there's no space both at the beginning and at the end.

Reply:
Why did you point it out :( That bothers me too, I just had the PSR-2 checker run automatically and it did this nonsense

Reply:
Now that you can't unsee, better fix it lol